R/ggcuminc.R

```diff
@@ -69,7 +69,7 @@ ggcuminc <- function(x, outcome = NULL,
     cli_inform("Plotting outcome {.val {outcome}}.")
   }
   if (any(!outcome %in% unique(df$outcome))) {
-    cli_abort("Argument {.code outcome} must be one or more of {.val {unique(df$outcome)}}")
+    cli_abort("Argument {.arg x} does not support {.cls survfit.coxphms} object.")
```

Comment:
I don't understand this change to the error message. Was this intentional?

Reply:
@ddsjoberg apologize that I may make this change by mistake. I have changed this section back to its original version (with **"Argument {.code outcome}"**)